examples/wandb_tracking/config.yaml

```diff
@@ -0,0 +1,22 @@
+name: wandb_example_pipeline
```

Comment:
for other examples you mentioned in the README the option to run with `-c config`, why not here?